cmd/check.go

```diff
@@ -25,7 +25,7 @@ var checkCmd = &cobra.Command{
 	// TODO: provide Short description for check command
 	Short: "Checks if operator meets minimum capability requirement.",
 	// TODO: provide Long description for check command
-	Long:  `The 'check' command checks if OpenShift operators meet minimum
+	Long: `The 'check' command checks if OpenShift operators meet minimum
```

Comment:
Can you remove this file from the commit. It's already added to the other PR and would create a Merge conflict.

Reply:
We'll need to rebase one of them anyway. So any conflicts can be managed.

Reply:
BTW just updated it! Removed the check file.